src/components/Dashboard/views/Typography/Typography.tsx

```diff
@@ -6,7 +6,7 @@ import Quote from '../../Typography/Quote';
 import Muted from '../../Typography/Muted';
 import Primary from '../../Typography/Primary';
 import Info from '../../Typography/Info';
-import Success from '../../Typography/Success';
+import { Success } from '../../../common/Typography/Success';
 import { Card, Danger, Warning } from '../../../common';
```

Comment:
they can be merged into one import statement

Reply:
this has been addressed